hack/dind-cluster.sh

```diff
@@ -367,13 +377,19 @@ start accepts the following options:
 
  -n [net plugin]   the name of the network plugin to deploy
 
+ -N                number of nodes in the cluster
+
  -b                build origin before starting the cluster
 
  -i                build container images before starting the cluster
 
  -r                remove an existing cluster
 
  -s                skip waiting for nodes to become ready
+
+Any of the arguments that would be used in creating openshift master can be passed
```

Comment:
Since these additional args are not applied directly to the dind binary, can we pass all these master args as '--additional-master-args'? At least this will allow us to expand more flags to dind in the future if needed